### PR TITLE
Enable autopath feature

### PR DESCRIPTION
**Pull request by r7vme**
I also increased memory limits - so it will allow to run up to 50K pods + services (as per [this page](https://github.com/coredns/deployment/blob/master/kubernetes/Scaling_CoreDNS.md#with-default-coredns-settings))

related to: https://github.com/giantswarm/giantswarm/issues/4662

### CODE REVIEW

File: helm/kubernetes-coredns-chart/templates/configmap.yaml [hunk 1]
```diff
@@ -31,4 +31,5 @@ data:
         {{- end }}
```
Comment (r7vme) [inline helm/kubernetes-coredns-chart/templates/configmap.yaml]: Do i also need to bump versions?

**Comment by r7vme** [comment]
PTAL

**Review by MarcelMue — APPROVED** [review]
LGTMIIWFY

**Comment by calvix** [comment]
should have this only in control plane for consistency?

**Review by calvix — APPROVED** [review]

File: helm/kubernetes-coredns-chart/templates/configmap.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/kubernetes-coredns-chart/templates/configmap.yaml]: No breaking changes, no need to bump version IMO.

**Review by TheoBrigitte — APPROVED** [review]
LGTM

**Review by tuommaki — APPROVED** [review]

**Pull request merged**